modules/connectivity/TrackStreamingStatus.ts

```diff
@@ -0,0 +1,711 @@
+import { getLogger } from '@jitsi/logger';
+
+import * as JitsiConferenceEvents from '../../JitsiConferenceEvents';
+import * as JitsiTrackEvents from '../../JitsiTrackEvents';
+import RTCEvents from '../../service/RTC/RTCEvents';
+import { createTrackStreamingStatusEvent } from '../../service/statistics/AnalyticsEvents';
+import JitsiConference from '../../types/hand-crafted/JitsiConference';
+import JitsiRemoteTrack from '../../types/hand-crafted/modules/RTC/JitsiRemoteTrack';
+import RTC from '../../types/hand-crafted/modules/RTC/RTC';
```

Comment:
I'm importing the hand-crafted types for `JitsiConference`, `JitsiRemoteTrack` and `RTC` here because I ran into issues with the auto generated ones. I want to limit the scope of this PR since it is already growing quite large

Reply:
I can also cast `rtc`, `conference`, and `track` to the `any` type for now if we shouldn't be importing these hand-crafted types